<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paaikkuna</class>
 <widget class="QMainWindow" name="Paaikkuna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>191</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>535</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Painoindeksilaskin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_sukupuoli">
      <property name="text">
       <string>Sukupuoli</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_arvo_sukupuoli">
      <property name="text">
       <string>arvo</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_paino">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Paino (kg)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_paino">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string> kg</string>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSlider" name="Slider_paino">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_pituus">
      <property name="text">
       <string>Pituus (cm)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_pituus">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string> cm</string>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSlider" name="Slider_pituus">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_painoindeksi">
      <property name="text">
       <string>Painoindeksi</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_arvo_painoindeksi">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_tulkinta">
      <property name="text">
       <string>Tulkinta</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLabel" name="label_arvo_tulkinta">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_paino</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_paino</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_pituus</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_pituus</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_paino</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_paino</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_pituus</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_pituus</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
